src/sg/gov/tech/geekout/1.0/geekout-open-attestation-document.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
we need to update the readme:
- add steps to create the oa related schema
- add steps to tell to create json example (and tell explicitely that any json file in src folder will be published)